examples/plotting/fading_between_maps.py

```diff
@@ -0,0 +1,59 @@
+"""
+Fading between two maps
+=======================
+
+Often it is useful to plot two maps of the Sun on top of each other, so
+features observed in one map (e.g. strong magnetic fields in a magnetogram)
+can be identified with features in the same place in another map
+(e.g. active regions in a EUV image).
+
+This example shows how to plot two maps on top of each other, with a slider to
+fade between them. Note that it assumes that the two maps are taken at the same
+time from the same observation point.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE, AIA_1600_IMAGE
+
+###############################################################################
+# Start by loading two AIA maps from the sample data.
+#
+# Note that these maps must be co-aligned. If they are taken at different times
+# or from different observation locations they will need to be aligned using
+# `~sunpy.map.GenericMap.rotate` or `reproject`.
```

Comment:
Does reproject link now?

Reply:
It will post merge of the reproject example PR